designs/2022-community-eslint-org/README.md

```diff
@@ -307,22 +349,6 @@ package), people could still use the old package without any extra problems
 
 - What exact acceptance criteria do we want to use?
```

Comment:
I think another question we should ask if the community core team is compensated out of the main ESLint budget? I can see arguments for both yes or no, and I’d. E interested in others thoughts.